packages/components/psammead-media-player/src/index.stories.jsx

```diff
@@ -3,10 +3,18 @@ import { storiesOf } from '@storybook/react';
 import { CanonicalMediaPlayer, AmpMediaPlayer } from '.';
 import { ampDecorator } from '../../../../.storybook/config';
 
+const withDuration = {
+  duration: '2:30',
+  durationSpoken: '2 minutes 30 seconds',
+  datetime: 'PT2M30S',
+};
+
 storiesOf('Components|Media Player', module).add('default', () => (
   <CanonicalMediaPlayer
     src="https://www.test.bbc.co.uk/ws/av-embeds/articles/c3wmq4d1y3wo/p01k6msp/en"
     placeholderSrc="https://ichef.bbci.co.uk/news/640/cpsdevpb/4eb7/test/ba7482d0-cca8-11e8-b0bf-f33155223fc4.jpg"
+    service="news"
+    mediaInfo={{ title: 'Dog chases cat.', ...withDuration }}
```

Comment:
Might be nice to add a story for audio as well as video, what does anyone else think?

Reply:
Don't we already have audio stories from L31 unless am missing something? 🤔

Reply:
Yes, I think we could add one for Audio with a placeholder. This is not the same as  'Audio Skin' from L31.